CHANGELOG.md

```diff
@@ -67,6 +76,7 @@ The main features of this release are:
 - #112 (ark-serialize) Make `bool`s checked serialization methods non-malleable.
 - #119 (ark-poly) Fix bugs in degree calculation if adding/subtracting same degree polynomials
      whose leading coefficients cancel.
+- #160 (ark-serialize, ark-ff, ark-ec) Support serializing when `MODULUS_BITS + FLAG_BITS` is greater than a multiple of 8 (fixes #47).
```

Comment:
errm, aren't all numbers greater than a multiple of 8 =p

Reply:
oh lol fair

```suggestion
- #160 (ark-serialize, ark-ff, ark-ec) Support serializing when `MODULUS_BITS + FLAG_BITS` is greater than the multiple of 8 just greater than `MODULUS_BITS` (fixes #47).
```

Reply:
that LGTM. Perhaps it should point out that this is the case for pallas/vesta curves?